native/src/routes/Pois.tsx

```diff
@@ -76,9 +79,47 @@ const Pois = ({
 }: PropsType): ReactElement => {
   const { t } = useTranslation('pois')
   const theme = useTheme()
+  const appState = useRef(AppState.currentState)
   const [selectedFeature, setSelectedFeature] = useState<Feature<Point> | null>(null)
   const [userLocation, setUserLocation] = useState<number[] | undefined>(undefined)
   const [featureLocations, setFeatureLocations] = useState<Feature<Point>[]>(prepareFeatureLocations(pois))
+  const [locationPermissionGranted, setLocationPermissionGranted] = useState<boolean>(false)
+
+  const onLocationPermission = useCallback(locationPermission => {
+    const permissionGranted = locationPermission === RESULTS.GRANTED
+    if (permissionGranted) {
+      Geolocation.getCurrentPosition(
+        (position: GeolocationResponse) => {
+          setUserLocation([position.coords.longitude, position.coords.latitude])
+          setLocationPermissionGranted(true)
+        },
+        (_error: GeolocationError) => setLocationPermissionGranted(false),
+        {
+          enableHighAccuracy: true,
```

Comment:
Same values are used in Landing.tsx i would propose to move them to a shared constant file